x-pack/plugins/uptime/public/components/monitor/__tests__/monitor_title.test.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import * as reactRouterDom from 'react-router-dom';
+import { MonitorPageTitle } from '../monitor_title';
+import { renderWithRouter, MountWithReduxProvider } from '../../../lib';
+import { AppState, store } from '../../../state';
+
+jest.mock('react-router-dom', () => ({
+  __esModule: true,
+  // @ts-ignore
```

Comment:
What's the reason for this being ignored? Any type script ignore statements should come with a comment explaining the why here. As a reviewer (or someone coming along later) it answers a lot of questions.